### PR TITLE
Fix running locally, fix PR links, style adjustments

### PR DESCRIPTION
**Pull request by oochelz**

### CODE REVIEW

File: app/assets/stylesheets/_settings.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@ $black: #0a0a0a;
 $white: #fefefe;
 $body-background: $white;
 $body-font-color: $black;
-$body-font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif;
+$body-font-family: 'Raleway', Helvetica, Roboto, Arial, sans-serif;
```
Comment (oochelz) [inline app/assets/stylesheets/_settings.scss]: All Raleway, all the time.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -25,10 +25,12 @@
     -webkit-box-shadow: 1px 1px 5px 2px rgba(221,221,221,1);
     -moz-box-shadow: 1px 1px 5px 2px rgba(221,221,221,1);
     box-shadow: 1px 1px 5px 2px rgba(221,221,221,1);
-    margin: 0px;
+    margin: 0;
+    margin-right: 1em;
```
Comment (oochelz) [inline app/assets/stylesheets/application.css]: Back up off my gravatar, man.

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -25,10 +25,12 @@
...
 }
 .commit {
-    margin-bottom: 1em;
+    padding-bottom: 1em;
+    padding-top: 1em;
```
Comment (oochelz) [inline app/assets/stylesheets/application.css]: Gimme some space.

File: app/controllers/pull_requests_controller.rb [hunk 4]
```diff
@@ -15,7 +15,7 @@ def create
     end
     begin
       github_api.pull_requests.create(user: params[:user_name], repo: params[:repo_name],
-                                      title: pr_titles.join(','), body: '',
+                                      title: pr_titles.join('; '), body: '',
```
Comment (oochelz) [inline app/controllers/pull_requests_controller.rb]: We frequently use semi-colons between PR names.

File: app/controllers/pull_requests_controller.rb [hunk 5]
```diff
@@ -63,9 +63,12 @@ def convert_to_pull_model(pull)
                     pull.user.avatar_url,
                     pull.title,
                     pull.number,
-                    pull.url.gsub('api.github','www.github'),
+                    pull.html_url,
```
Comment (oochelz) [inline app/controllers/pull_requests_controller.rb]: API and site do not use the same URL structure.

File: app/controllers/pull_requests_controller.rb [hunk 6]
```diff
@@ -63,9 +63,12 @@ def convert_to_pull_model(pull)
...
                     pull.created_at.in_time_zone('Eastern Time (US & Canada)').strftime('%m/%d/%Y | %I:%M %p EST'),
-                    "#{info[4]}/#{info[5]}")
+                    pull.head.repo.name)
```
Comment (oochelz) [inline app/controllers/pull_requests_controller.rb]: Use the repo name, not the URL parts.

File: app/views/pull_requests/index.html.haml [hunk 7]
```diff
@@ -18,12 +18,8 @@
...
 - if controller.show_pr_section?
   .row.commit
-    .small-12.large-3.columns
-      By:
-    .small-12.large-6.columns.deployment-message
-      Repo / Number / Title 
-    .small-12.large-3.columns.end-of-header
-      Date Opened
+    .small-12.columns
+      Author / Repo / Title / Date Opened
```
Comment (oochelz) [inline app/views/pull_requests/index.html.haml]: Match other page.

File: app/views/pull_requests/index.html.haml [hunk 8]
```diff
@@ -31,8 +27,9 @@
       .small-12.large-3.columns
         = image_tag(pull.user_avatar, class: 'gravatar')
         = link_to pull.user_login, "https://www.github.com/#{pull.user_login}"
-      .small-12.large-6.columns.deployment-message
+      .small-12.large-3.columns.deployment-message
         = pull.repo_name
+      .small-12.large-3.columns.deployment-message
```
Comment (oochelz) [inline app/views/pull_requests/index.html.haml]: More spacing.

File: config/environments/development.rb [hunk 9]
```diff
@@ -13,15 +13,9 @@
...
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
-  # Raise an error on page load if there are pending migrations.
-  config.active_record.migration_error = :page_load
-
```
Comment (oochelz) [inline config/environments/development.rb]: These were causing issues when running locally. Not using action mailer or active record at all.

File: app/assets/stylesheets/_settings.scss (hunk 1 above)
Comment (bowersaa) [inline app/assets/stylesheets/_settings.scss]: :yellow_heart:

File: app/views/pull_requests/index.html.haml (hunk 7 above)
Comment (bowersaa) [inline app/views/pull_requests/index.html.haml]: Which other page are you trying to match? There are both ways in the pages now. (I'd prefer for the other page to match this one's previous way)

Comment (bowersaa) [inline app/views/pull_requests/index.html.haml]: @oochelz @usbsnowcrash

Comment (oochelz) [inline app/views/pull_requests/index.html.haml]: Oh, when I had it over each, the small screen looked stupid. The "other page" is Deployments.

**Pull request merged**